R/tidytabulardata.R

```diff
@@ -80,6 +81,14 @@ TidyTabularData <- function(
     if (hide.empty.rows.and.columns)
         x <- HideEmptyRowsAndColumns(x)
 
+    # This is only needed for outputs in Q
+    # This is because RemoveRowsAndOrColumns and
+    # HideEmptyRowsAndColumns call copyAttributesIfNotQTable
+    # However by default verbs has AllowQTableSubscripting
+    # set to false in Q
+    if (!is.null(statistic.attribute))
+        attr(x, "statistic") <- statistic.attribute
+
```

Comment:
This isn't a problem for the visualization Standard R outputs because they set AllowQTableSubscripting: https://github.com/Displayr/Plugins/blob/76a00224a11cb658ddb221048dbe7271028e38e0/src/Standard%20R/Visualization/Area/Area.R#L14